apps/trading/pages/portfolio/index.page.tsx

```diff
@@ -10,76 +10,88 @@ import classNames from 'classnames';
 import type { ReactNode } from 'react';
 import { VegaWalletContainer } from '../../components/vega-wallet-container';
 import { DepositsContainer } from './deposits-container';
+import { Allotment, LayoutPriority } from 'allotment';
+import 'allotment/dist/style.css';
```

Comment:
chore (blocking): I think this might include the styles twice in the css bundle. Can you create a an allotment component in ui-toolkit and import it there once?

Reply:
Have now added a component.